src/viewer.js

```diff
@@ -965,10 +967,14 @@ class VolumeImageViewer {
       tileSizes: this[_pyramid].tileSizes
     })
 
+    const resolutions = this[_options].disableResolutionsArray
```

Comment:
Instead of disabling it, I prefer exposing the map view resolutions in case someone wants to define the value. You can pass null If you're going to avoid the default value:

```
@param {number[]} [options.mapViewResolutions] Map's view list of resolutions. If not passed, the tile grid resolution will be used.
...
    let mapViewResolutions = this[_tileGrid].getResolutions();
    if (this[_options].mapViewResolutions !== null) {
      mapViewResolutions = this[_options].mapViewResolutions
    } 
    ...
     if (this[_options].highlightColor == null) {
      this[_options].highlightColor = [140, 184, 198]
    }
```

Please double check that everything works with the new approach.

Reply:
I had to change this a little bit to keep the originality of the function.
Take a look on my code suggestion. It keeps working as always and give the user the option to use the mapViewResolutions.

<img width="815" alt="Captura de Tela 2023-08-21 às 16 43 11" src="https://github.com/ImagingDataCommons/dicom-microscopy-viewer/assets/115191433/0825bf48-c6ac-404c-ad85-5e47e2fdffae">